examples/surrounding_nitrogen_bonds/nn_bond_length_histogram.py

```diff
@@ -0,0 +1,108 @@
+"""
+Script for generating a histogram based on the bond length of
+single nitrogen-nitrogen bonds
+Usage:
+    python nn_bond_length_histogram.py --smiles_database [DATABASE] --fingerprinted [T or F]
+"""
+
+from openeye import oechem, oeomega, oequacpac, oeff
+import argparse
+import matplotlib.pyplot as plt
+import math
+
+def generateHistogram():
+    """
+    Generates a bond length histogram for single nitrogen-nitrogen bonds
+    using a .smi file
+    """
+    
+    parser = argparse.ArgumentParser()
```

Comment:
Might be better to put the argument parsing outside the function (i.e. in a `main()` function or at the beginning of `if __name__ == "__main__"`. That would make this function more portable.